lib/data/book.js

```diff
@@ -22,7 +22,9 @@ module.exports = (h = {}, args = {}) => ({ state = {}, data = {} } = {}) => {
     : new OrderBook(original, raw)
 
   if (books[key]) {
-    nextBook.updateWith(original)
+    const entries = Array.isArray(original[0]) ? original : [original]
```

Comment:
given that the first element is indeed an array, why should we discard all the other elements and use just the first one? Why not flatten it?

Reply:
because if the first element of `original` array is an array (order book - an array with 3 numeric elements) then all other elements are also an array

Reply:
there will be cases where original is an empty array as well (example for testing: in staging(SAN/USD PAIR)) which would result in entries to be [ [] ]

Reply:
fixed